Makefile

```diff
@@ -2,17 +2,19 @@ CXX = g++
 
 BUILD_DIR = ./build/
 INC_DIR = ./include/
+INC_LIB = ./lib/
```

Comment:
What are the underlying semantics of `INC_DIR` and `INC_LIB`? (Also `LIB_DIR`)

Reply:
Sorry for being picky!

Reply:
`INC_DIR` holds the path to the include directory. As far as I understood it, this is would hold public headers, but as the xarm-commander won't be used as a library, I guess this folder won't be even used..
`INC_LIB` holds the path to the /lib directory which holds all external libraries (not sure if /external should be used for such submodules like the CLI11)
`LIB_DIR` holds the path to the build folder of the xarm SDK which includes the compiled binaries for the xarm-SDK
So the `INC_DIR` may be removed in future versions..

Reply:
According to the following sources:
- https://raw.githubusercontent.com/vector-of-bool/pitchfork/develop/data/spec.bs
- https://softwareengineering.stackexchange.com/questions/81899/how-should-i-organize-my-source-tree

`./lib` should then be `./libs` and `$LIB`, `INC_DIR` can go, and `LIB_DIR` can be more specific `XARM_SDK_LIB` or something?

Reply:
Yes, you're right!
I'll change that :)